src/util/strip.js

```diff
@@ -1,5 +1,14 @@
 /** Strip HTML tags, convert nbsp to normal spaces, and trim. */
-export const strip = html => html
-  .replace(/<(?:.|\n)*?>/gm, '')
-  .replace(/&nbsp;/gm, ' ')
-  .trim()
+export const strip = (html, stripFormatting = true) => {
```

Comment:
Since you are using eS6 arrow functions, you really don't need this return key word

`export const strip = (html, stripFormatting = true) => stripFormattong ?       html
        .replace(/<(?:.|\n)*?>/gm, '')
        .replace(/&nbsp;/gm, ' ')
        .trim() :  html
      .replace(/<(?:.|\n)*?([^bui])>/gm, '')
      .replace(/&nbsp;/gm, ' ')
      .trim()`
What do you think?